xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -458,6 +492,18 @@ static int PlayMedia(const std::vector<std::string>& params)
   if (!item.m_bIsFolder && item.IsPlugin())
     item.SetProperty("IsPlayable", true);
 
+  bool restricted{false};
+  if (!item.m_bIsFolder && !item.IsPlayList())
+  {
+    if (item.IsAudio())
+      restricted = IsVideoOrAudioRestricted(item, "audio");
```

Comment:
You can probably just shorten this whole block.

```suggestion
      if (IsVideoOrAudioRestricted(item))
        return false:
```